src/ds/d400/d400-nonmonochrome.cpp

```diff
@@ -29,13 +29,13 @@ namespace librealsense
         auto pid = dev_info->get_group().uvc_devices.front().pid;
         auto& depth_ep = get_depth_sensor();
 
-        // RGB for D455/D465 from Left Imager is available with FW 5.12.8.100
-        if ((val_in_range(pid, { RS455_PID , RS465_PID })) && (_fw_version < firmware_version("5.12.8.100")))
+        // RGB for D455 from Left Imager is available with FW 5.12.8.100
+        if ((val_in_range(pid, { RS455_PID })) && (_fw_version < firmware_version("5.12.8.100")))
             return;
 
-        if ((_fw_version >= firmware_version("5.5.8.0")) && (!val_in_range(pid, { RS_USB2_PID, RS465_PID })))
+        if ((_fw_version >= firmware_version("5.5.8.0")) && (!val_in_range(pid, { RS_USB2_PID })))
         {
-            if (!val_in_range(pid, { RS405_PID , RS455_PID, RS465_PID }))
+            if (!val_in_range(pid, { RS405_PID }))
```

Comment:
Did you mean to remove D455 here?

Reply:
No.. thanks for catching that, will fix now

Reply:
Fixed and pushed